src/config/index.js

```diff
@@ -62,11 +63,18 @@ const config = new Map();
 
   config.set('CSP', CSP);
 
-  const APP_NAMES = ['search', 'disco'];
-  if (APP_NAMES.indexOf(process.env.APP_NAME) < 0) {
-    config.set('currentApp', 'core');
+  // This is the list of apps allowed to run.
+  const validAppNames = ['search', 'disco'];
+  config.set('validAppNames', validAppNames);
+
+  // Create a list of apps to build targets for.
+  const appName = process.env.APP_NAME;
+  if (validAppNames.indexOf(appName) > -1) {
```

Comment:
Yep good call - fixed in https://github.com/mozilla/addons-frontend/commit/29c65944414aa15ed75742b7142935e4be54826d